src/dirs.proj

```diff
@@ -4,6 +4,7 @@
 
   <ItemGroup>
     <Project Include="*\**\*.csproj" />
+    <Project Include="*\**\*.depproj" />
```

Comment:
I used a depproj because it was the way of getting the dnxcore 5.0 dependencies deployed as part of the package since that runs for full framework and for .Net Core. I'll take a look at how you do it, and submit a PR for this.